perllib/FixMyStreet/Integrations/ExorRDI.pm

```diff
@@ -43,13 +43,23 @@ sub construct {
     my $cobrand = FixMyStreet::Cobrand->get_class_for_moniker('oxfordshire')->new;
     my $dtf = $cobrand->problems->result_source->storage->datetime_parser;
 
+    my $missed_cutoff = DateTime->now - DateTime::Duration->new( hours => 24 );
     my %params = (
         -and => [
             state => [ 'action scheduled' ],
             external_id => { '!=' => undef },
-            'admin_log_entries.action' => 'inspected',
-            'admin_log_entries.whenedited' => { '>=', $dtf->format_datetime($self->start_date) },
-            'admin_log_entries.whenedited' => { '<=', $dtf->format_datetime($self->end_date) },
+            -or => [
+                -and => [
+                    'admin_log_entries.action' => 'inspected',
+                    'admin_log_entries.whenedited' => { '>=', $dtf->format_datetime($self->start_date) },
+                    'admin_log_entries.whenedited' => { '<=', $dtf->format_datetime($self->end_date) },
+                ],
+                -and => [
+                    extra => { -not_like => '%rdi_processed%' },
```

Comment:
As well as the "probably don't want to do this when admin page", is there a performance concern here? I guess it is only the one cobrand + state and date, so probably not?